lib/app/admin/data-reset.js

```diff
@@ -0,0 +1,212 @@
+var q = require('q');
+var async = require('async');
+var _ = require('lodash');
+// default list of types of objects to remove, can be overriden when passed in request body
+// order of the list is important for building relations between them, parent items should be listed first
```

Comment:
Probably good to include an example here. 

E.g. workorders have references to work flows using the workflow IDs. Therefore, the work flows should be created first.